### ISSUE TITLE
serenity report : stackoverflow error

### ISSUE THREAD

**Issue by mmanach** [serenity-bdd/serenity-core#972]
Hello,

- serenity core / gradle : 1.6.5
- serenity jbehave : 1.30
- Windows 10 64 bits

Since upgrading serenity core / gradle from 1.5.9 and jbehave from 1.27, I have this error log when I execute
```
gradle clean test aggregate
```
```
        at net.thucydides.core.reports.html.Breadcrumbs$BreadcrumbsBuilder.fromTagsIn(Breadcrumbs.java:32)
SOME REPORT PAGES COULD NOT BE GENERATED
 * Failed to generate report for TagReportingTask for TestTag{name='Logout/Logout', type='story'} - java.util.concurrent.ExecutionException: java.lang.StackOverflowError
net.thucydides.core.reports.html.Breadcrumbs$BreadcrumbsBuilder.elementsOf(Breadcrumbs.java:43)
```
(but the build is green)

It seems that the reason of this error is in my logout.story : 
```
Meta:
@issue #TPSVC-3373
````

When I remove the '@issue' tag, there is no longer this stacktrace.

**Comment by wakaleo** [comment]
Can you provide a sample project?

**Comment by mmanach** [comment]
Yep I will tomorrow. Seems that the same name for folder and story is involved too.

**Comment by mmanach** [comment]
https://github.com/mmanach/serenity-issue-972

So it seems that the cause is a combination of same folder name / story name and story meta tag "@issue"

**Comment by wakaleo** [comment]
Should be fixed in 1.6.6-rc.1
